content/applications/marketing/events/sell_tickets.rst

```diff
@@ -1,68 +1,137 @@
+
+The :guilabel:`Online Ticketing` setting allows users to sell event tickets online through their
+integrated Odoo website.
+
+To activate a setting, tick the checkbox beside the desired feature's label, and click
+:guilabel:`Save` to finish enabling it.
 
 .. note::
-   If these options are *not* enabled, a default :guilabel:`Register` button will be available for
-   free registrations.
+   If these options are *not* enabled, a default :guilabel:`Register` button becomes available for
+   visitors to interact with and procure free registrations to the event.
 
 .. image:: sell_tickets/events-settings-tickets.png
    :align: center
    :alt: View of the settings page for Odoo Events.
 
-Sell tickets through sales orders
-=================================
+With those settings enabled, Odoo automatically creates a new *Product Type* called, *Event Ticket*,
+which is accessible on every product form. Odoo also creates three event registration products (with
+the *Product Type* set to *Event Ticket*) that can be used or modified for event tickets.
+
+.. important::
+   When creating a new event registration product, the *Product Type* **must** be set to *Event
+   Ticket* on the product form, in order for it to be selected in the *Product* column under the
+   *Tickets* tab on an event form.
+
+   .. image:: sell_tickets/events-tickets-registration-product.png
+      :align: center
+      :alt: View of an event form highlighting the column product under the tickets tab in Odoo.
+
+.. note::
+   Any event with paid tickets sold will feature a :guilabel:`Sales` smart button at the top of the
+   event form, where the respective sales orders attributed to those ticket sales become available.
+
+   .. image:: sell_tickets/events-sales-smartbutton.png
+      :align: center
+      :alt: View of an event's form and the sales smart button in Odoo Events.
+
+   Clicking that smart button reveals a separate page, showcasing all the sales orders (standard
+   and/or online) related to tickets that have been sold for that specific event.
 
-In the :guilabel:`Sales` application, choose a previously-created Event Registration (as if it were
-a product), and add it as a product line. Upon adding the registration, a pop-up appears, which
-allows for the selection of a specific event (and ticket tier). That specific event ticket is then
-attached to the sales order.
+Sell event tickets with the Sales app
+=====================================
 
-.. image:: sell_tickets/events-through-sales-order.png
+To sell event tickets with sales orders, start by navigating to the :menuselection:`Sales` app.
+Then, click :guilabel:`New` to open a new quotation form.
+
+After filling out the top portion of the form with the appropriate customer information, click
+:guilabel:`Add a product` in the :guilabel:`Order Lines` tab. Then, select (or create) an event
+registration product configured with its *Product Type* set to *Event Ticket* on its product form.
```

Comment:
Specify the Product column, just to be precise. And since you stated "(or create)" I think it is safe to use :guilabels: for the field and value on the product form :)
```suggestion
:guilabel:`Add a product` in the :guilabel:`Order Lines` tab. Then, in the :guilabel:`Product`
column, select (or create) an event registration product configured with its :guilabel:`Product
Type` set to :guilabel:`Event Ticket` on its product form.
```